tools/depends/native/binutils-native/Makefile

```diff
@@ -0,0 +1,38 @@
+include ../../Makefile.include
+PLATFORM=$(NATIVEPLATFORM)
+DEPS= ../../Makefile.include Makefile readelfOnly.patch
+
+# lib name, version
+LIBNAME=binutils
+VERSION=2.29
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+BASE_URL=https://ftp.gnu.org/gnu/binutils
```

Comment:
So should we add an inline Todo or will somebody just do it? :)

Reply:
Gimme a min...

Reply:
@Razzeee either someone has beaten me to it - or it is available on our mirrors since aug 5th already - see: http://mirrors.kodi.tv/build-deps/sources/

Reply:
done it a few minutes ago

Reply:
Should I add the link to the BASE_URL?

Reply:
just remove the BASE_URL line, http://mirrors.kodi.tv/build-deps/sources is the default